### PR TITLE
initial unit tests for dice loss

### LINKED ISSUE #10: Dice loss function

### PR DESCRIPTION
**Pull request by wyli**
Dice loss is already implemented in the repo and should be enough for the MVP. 
This PR provides a gentle unit test for the module.
closes #10.

next steps:
- might need to check input data types
- might need sparse tensor for the one-hot encoding
- would be great to have generic implementations such as https://weisslab.cs.ucl.ac.uk/FernandoPerezGarcia/episurg/blob/master/episurg/loss.py @fepegar

### CODE REVIEW

File: tests/test_dice_loss.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                'include_background': True,
+            },
+            {
+                'pred': torch.tensor([[[[1., -1.], [-1., 1.]]], [[[1., -1.], [-1., 1.]]]]),
+                'ground': torch.tensor([[[[1., 1.], [1., 1.]]], [[[1., 0.], [1., 0.]]]]),
+                'smooth': 1e-4,
+            },
+            0.416636,
+        ],
+    ])
```
Comment (ericspod) [inline tests/test_dice_loss.py]: This looks like a good way to write multiple tests using the same code, so long as it plays well with the output from unittest when they fail. Perhaps put this list into a separate variable above the test class however just so the @parameterized tag doesn't get lost.
